docs/platform/release/full-changelog.md

```diff
@@ -190,362 +190,362 @@ Versions marked with a number and date (e.g. Falcon Platform v2.1.1 (2020-05-08)
+- Breaking change: Falcon E-commerce UI Kit has been discontinued, instead we introduce [Falcon Front Kit](#falcon-front-kit-v1.0.0), [Falcon UI Kit](#falcon-ui-kit-v1.0.0), [Falcon Data](#falcon-data-v1.0.0), [Falcon Shop Data](#falcon-shop-data-v1.0.0) and [Falcon Blog Data](#falcon-blog-data-v1.0.0), (see rework [log](https://github.com/deity-io/falcon/blob/a5274623c148b9b0c8cc1d17254d7c89395955da/packages/falcon-ecommerce-uikit/uikit.md))
 
 ### Falcon Front Kit
 
 - initial release
-- added convention for validation error messages translation ([#585](https://github.com/deity-io/falcon/pull/585))
-- fixed synchronization of account and sign in icon with customer logging in and out state ([#521](https://github.com/deity-io/falcon/pull/521))
-- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`) ([#596](https://github.com/deity-io/falcon/pull/596))
-- improved dynamic url resolution ([#618](https://github.com/deity-io/falcon/pull/618))
-- added `Add`/`EditAddressFormProvider` components to handle relevant form data ([#624](https://github.com/deity-io/falcon/pull/624))
-- added `Submit` component which provides context to form submit components ([#548](https://github.com/deity-io/falcon/pull/548))
-- fixed `LocaleSwitcher` which did not take into account `whitelist`ed languages, and `LocaleContext` which hardcoded language fallback instead of taking it from `clientConfig` ([#698](https://github.com/deity-io/falcon/pull/698))
-- extracted `CurrencyProvider` and `Currency` components from `LocaleProvider` and `Locale`, also introduced `useLocale` and `useCurrency` hooks ([#698](https://github.com/deity-io/falcon/pull/698))
-- updated Formik to version 2.0 ([#705](https://github.com/deity-io/falcon/pull/705))
-- added `InBrowserOnly` component ([#728](https://github.com/deity-io/falcon/pull/728))
-- added `SwitchDynamicURL` component in order to define dynamic URL routes in `react`/`react-router` `v4` way ([#738](https://github.com/deity-io/falcon/pull/738))
-- added `SEO` component to render `title`, `description` and `keywords` as proper meta-tags ([#743](https://github.com/deity-io/falcon/pull/743))
-- added support for url with query string provided to `<ProtectedRoute />` component via `redirectTo` prop ([#765](https://github.com/deity-io/falcon/pull/765))
+- added convention for validation error messages translation
+- fixed synchronization of account and sign in icon with customer logging in and out state
+- added Apollo hooks support (upgrade `apollo-client` to version `2.6.x`)
+- improved dynamic url resolution
+- added `Add`/`EditAddressFormProvider` components to handle relevant form data
+- added `Submit` component which provides context to form submit components
+- fixed `LocaleSwitcher` which did not take into account `whitelist`ed languages, and `LocaleContext` which hardcoded language fallback instead of taking it from `clientConfig` falcon/pull/698))
```

Comment:
```suggestion
- fixed `LocaleSwitcher` which did not take into account `whitelist`ed languages, and `LocaleContext` which hardcoded language fallback instead of taking it from `clientConfig`
```